modules/exploits/windows/smb/smb_relay.rb

```diff
@@ -83,7 +83,8 @@ module is not able to clean up after itself. The service and payload
           ['MSB', 'MS08-068'],
           ['URL', 'http://blogs.technet.com/swi/archive/2008/11/11/smb-credential-reflection.aspx'],
           ['URL', 'https://en.wikipedia.org/wiki/SMBRelay'],
-          ['URL', 'http://technet.microsoft.com/en-us/sysinternals/bb897553.aspx']
+          ['URL', 'http://technet.microsoft.com/en-us/sysinternals/bb897553.aspx'],
+          ['ATT&CK', Mitre::Attack::Technique::T1021_002_SMB_WINDOWS_ADMIN_SHARES]
```

Comment:
Should we be manually adding the parent `T1021_REMOTE_SERVICES` here implicitly behind the scenes or explicitly? 🤔

```suggestion
          ['ATT&CK', Mitre::Attack::Technique::T1021_REMOTE_SERVICES],
          ['ATT&CK', Mitre::Attack::Technique::T1021_002_SMB_WINDOWS_ADMIN_SHARES]
```

Reply:
In answer to this, see `exploits/linux/misc/igel_command_injection`
It works through _either_ telnet _or_ VNC.  VNC has a special subclause, but telnet does not.  The only way to express both is with the overarching technique and the sub-technique.  It might be useful to keep them separate so that you can see both?